mkdocs.yml

```diff
@@ -110,6 +110,7 @@ nav:
       - 'Compatibility Presets': 'presets-compatibility.md'
       - 'npm Presets': 'presets-npm.md'
       - 'Regex Manager Presets': 'presets-regexManagers.md'
+      - 'Replacement Presets': 'presets-replacements.md'
```

Comment:
I've confirmed this works in Gitpod. 🥳 

I'm not sure about the best placement for this item though. We don't sort the list items alphabetically right now. Might be nice to sort all items from A through Z?

Reply:
Alphabetical sounds good.

Reply:
I'll open a new PR to sort the preset sidebar items alphabetically after this PR is merged.